app/views/api/v1/commodities/show.json.rabl

```diff
@@ -5,6 +5,13 @@ attributes :producline_suffix, :description, :number_indents,
            :goods_nomenclature_item_id, :bti_url, :formatted_description,
            :description_plain, :consigned, :consigned_from, :basic_duty_rate
 
+  footnotes = (@commodity.footnotes + @commodity.heading.footnotes).uniq
```

Comment:
We could probably refactor this similar to the uptree method - but that could be done later.

Reply:
@matthewford agreed, good point